.env.prod

```diff
@@ -1,2 +1,3 @@
+APP_ENV=prod
```

Comment:
This file is only loaded when the env = prod, so it's redundant.

Reply:
@vvasiloi I forgot to remove 🤗